src/components/Modal/index.tsx

```diff
@@ -141,13 +143,13 @@ export default function Modal({
                       style: { transform: y.interpolate((y) => `translateY(${(y as number) > 0 ? y : 0}px)`) },
                     }
                   : {})}
-                aria-label="dialog content"
-                minHeight={minHeight}
-                maxHeight={maxHeight}
-                mobile={isMobile}
-                scrollOverlay={scrollOverlay}
-                hideBorder={hideBorder}
-                maxWidth={maxWidth}
+                aria-label="dialog"
+                $minHeight={minHeight}
+                $maxHeight={maxHeight}
+                $isBottomSheet={isBottomSheet}
+                $scrollOverlay={scrollOverlay}
+                $hideBorder={hideBorder}
+                $maxWidth={maxWidth}
```

Comment:
While I wasn't necessarily happy with the prefix when I first saw it, I think it's a really good differentiator between props for the styled component vs actual props for the component (they can have impact on styling too, but have actual impact on the behaviour)

Reply:
yeah i think this is fine